src/components/TextField/TextField.scss

```diff
@@ -100,9 +100,7 @@ $stacking-order: (
   padding: control-vertical-padding() $backdrop-horizontal-spacing;
   background: none;
   border: border(transparent);
-  font-family: inherit;
-  font-size: inherit;
-  font-weight: inherit;
+  font-family: font-family();
```

Comment:
I am sad we need this... makes sense knowing we don't put font family on parent elements.